server/controllers/loginLogicHandler.js

```diff
@@ -0,0 +1,50 @@
+const { compare } = require('bcryptjs');
+const { signAuthentication, customizedError } = require('../auth');
+const { getUserByEmail } = require('../database/queries');
+const { loginSchema } = require('../utils');
+
+const loginLogicHandler = (req, res, next) => {
+  const { email, password } = req.body;
+  let id;
+  let username;
+  loginSchema
+    .validateAsync(req.body)
+    .then(() => getUserByEmail(email))
+    .then((data) => {
+      if (!data.rows.length) {
+        customizedError({
+          errorMessage: "Email doesn't exists, Try another one or sign up",
+          status: 401,
```

Comment:
When the user logs in with an invalid email, Is this the right status code to use?